script.js

```diff
@@ -1 +1,125 @@
-//CEOS 19기 프론트엔드 파이팅🔥 ദ്ദി˶ˊᵕˋ˵)
+const inputField = document.querySelector('.todo-input'); //todo 입력창
+const TodoButton = document.querySelector('.todo-input-button'); //+추가 버튼
+const todoLists = document.querySelector('.todo-lists'); //todo list목록
+const doneLists = document.querySelector('.done-lists'); //done list목록
+const date = document.querySelector('h1#date'); //날짜 표기
+const week = document.querySelector('h3#week'); //요일 표기
+const showTodoNumber = document.querySelector('.show-todo-number'); //todo list div 개수
+const showDoneNumber = document.querySelector('.show-done-number'); //done list div 개수
+
+
+//lists 개수 세기
+function todoListsNumber(){
+    const todoNumber = todoLists.getElementsByTagName("div");
+    const doneNumber = doneLists.getElementsByTagName("div");
+    
+    showTodoNumber.innerText = `${todoNumber.length} lists to do`;
+    showDoneNumber.innerText = `${doneNumber.length} lists are done! Way to go : )`
+}
```

Comment:
```suggestion
function todoListsNumber(){
    const todoNumber = todoLists.querySelectorAll("li").length;
    const doneNumber = doneLists.querySelectorAll("li").length;
    
    showTodoNumber.innerText = `${todoNumber.length} lists to do`;
    showDoneNumber.innerText = `${doneNumber.length} lists are done! Way to go : )`
}
```

.todo-list라는 선택자로 가져온 값의 하위 div의 개수를 세기보다는, 하위의 li 태그를 세는 게 더 직관적인 코드가 될 거 같아요~!